src/pages/Profile.js

```diff
@@ -473,7 +473,7 @@ const NotificationSettingsTab = ({ setSaving }) => {
             `notifications/AbortedSubmissionNotification/${label}`
           ]
         })
-      })(workspaces)
+      })(_.filter({ public: false }, workspaces)) // public workspaces are not useful to display here and clutter the list
```

Comment:
I think using `_.flow` here would make this slightly more conventional and readable.

```js
_.flow(
  _.filter(...),
  _.map(...)
)(workspaces)
```

Reply:
Done in 75645a4b